overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -125241,7 +125247,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Get Stone(d)!",
-          "desc:8": "Place any 8 Dimensional Stone types near your Daisy and wait 30 seconds to obtain Livingrock!\n\nYou definitely want to automate this with ME Annihilation and Formation Planes."
+          "desc:8": "Place any 8 Dimensional Stone types near your Daisy and wait 30 seconds to obtain Livingrock!\n\nYou definitely want to automate this with ME Annihilation and Formation Planes or similar."
```

Comment:
`use X and Y or similar` should either be dropped or change to `something such as X and Y`